modules/andes-core/management/common/src/main/java/org/wso2/andes/management/common/mbeans/SubscriptionManagementInformation.java

```diff
@@ -46,7 +46,17 @@
             @MBeanOperationParameter(
                     name = "destinationType",
                     description = "Destination type of the subscriptions") String destinationType)
-    throws MBeanException;
+            throws MBeanException;
+
+    /**
+     * Get the pending message count for the specified subscription
+     * @param queueName for which the pending message cound need to be calculated
```

Comment:
Let's write this according to the java doc format. See http://www.oracle.com/technetwork/articles/java/index-137868.html